src/content/blog/2024/fastquorum_intro.mdx

```diff
@@ -0,0 +1,301 @@
+---
+title: 'nf-core/fastquorum introduce'
+subtitle: 'The nf-core fgbio Best Practices FASTQ to Consensus Pipeline'
+pubDate: 2024-05-27T00:19:00+01:00
+authors:
+    - 'nh13'
+label:
+    - 'pipeline'
+    - 'bioinformatics'
+    - 'genomics'
+    - 'fgbio'
+    - 'fastquorum'
+---
+
+import { YouTube } from '@astro-community/astro-embed-youtube';
+
+import but_why from '@assets/images/blog/fastquorum/but_why.gif';
+import i_want_the_truth from '@assets/images/blog/fastquorum/i_want_the_truth.gif';
+import darwin from '@assets/images/blog/fastquorum/darwin.png';
+import duplex_sequencing_schema from '@assets/images/blog/fastquorum/duplex_sequencing_schema.png';
+import explaining_umis from '@assets/images/blog/fastquorum/explaining_umis.png';
+import bioconda_download from '@assets/images/blog/fastquorum/bioconda_download.png';
+import big_deal from '@assets/images/blog/fastquorum/big_deal.jpg';
+import fastquorum_diagram from '@assets/images/blog/fastquorum/fastquorum_diagram.png';
+import fulcrumgenomics_logo from '@assets/images/blog/fastquorum/fulcrumgenomics.svg';
+
+import { Image } from 'astro:assets';
+
+We are thrilled to announce the first release of [`nf-core/fastquorum`][nf-core-fastquorum-link] pipeline, which implements the [fgbio Best Practices FASTQ to Consensus Pipeline][fgbio-best-practices-link] to produce consensus reads using unique molecular indexes/barcodes (UMIs).
+Developed by [Nils Homer][nilshomer-linkedin-link] at [Fulcrum Genomics][fulcrum-genomics-link], the pipeline utilizes the [`fgbio` Bioinformatics toolkit][fgbio-link] to enable producing ultra-accurate reads for low frequency variant detection.
```

Comment:
Mightbe worth mentioning here you're talking about genomics/DNA sequencing, given we have increasingly more non-genomics people in nf-core

Reply:
You sort of do this nicely at the 'fgbio: the ...' section